README/DEV_DEPLOY.md

```diff
@@ -1,34 +1,38 @@
-## Hypothetical Deploying from Dev
+## Deploying from Dev
 
 ### Background
 
-As of 31 May 2024, SocialPredict does not have a production version launched. That is to say, SocialPredict is designed for development, meaning out of the box, everything it comes with can be worked on within a local machine, but it's not set up to be served securely as a production instance on the web that can't be easily hacked.
+As of 28 July 2024, SocialPredict does not have a production version launched. SocialPredict is designed for development: users can deploy software out of the box. Everything it comes with can be worked on within a local machine, but it's not set up to be served securely as a production instance on the web that can't be easily hacked.
```

Comment:
SocialPredict has a production version launched, though the production version does not serve files in an ideal way. E.g. frontend files are not served statically via Nginx, but rather a node docker container serves the files. The backend binary golang is written recursively, designed to be mounted as a dev mode container, rather than just being running inside of a backend container. Moreover, there are lots of potential security flaws.